eureka2-client/src/main/java/com/netflix/eureka2/client/transport/ResolverBasedTransportClient.java

```diff
@@ -76,7 +79,7 @@ public MessageConnection call(
                                             ObservableConnection<Object, Object> conn) {
                                         return new SelfClosingConnection(
                                                 new HeartBeatConnection(
-                                                        new BaseMessageConnection("client", conn, metrics),
+                                                        new BaseMessageConnection(clientId, conn, metrics),
```

Comment:
If this is purely for testing purposes, it might be useful to define a counting id in the client, and append the counting id to the end of this to differentiate between channel generations.